### PR TITLE
Update nightly ORNL versions and README.md

### PR DESCRIPTION
**Pull request by prckent**
## Proposed changes

Update scripts and versions used in nightlies. These latest versions have been used for a couple of days and have no major build issues.

Update and improve project README

## What type(s) of changes does this code introduce?

- Testing changes (e.g. new unit/integration/performance tests)
- Documentation changes

### Does this introduce a breaking change?

- No

## What systems has this change been tested on?

sulfur, nitrogen in nightlies

## Checklist

- Yes. This PR is up to date with current the current state of 'develop'
- Yes. Documentation has been added (if appropriate)

### CODE REVIEW

**Comment by prckent** [comment]
Test this please

File: README.md [hunk 1]
```diff
@@ -100,10 +108,10 @@ cmake ..
...
 ```
 
- The complexities of modern computer hardware and software systems are
- such that you should check that the auto-configuration system has made
- good choices and picked optimized libraries and compiler settings
- before doing significant production. i.e. Check the details below.
+The complexities of modern computer hardware and software systems are
+such that you should check that the auto-configuration system has made
+good choices and picked optimized libraries and compiler settings
+before doing significant production. i.e. Check the details below. For MPI builds we recommend
```
Comment (markdewing) [inline README.md:114]: Sentence is incomplete?

File: README.md [hunk 2]
```diff
@@ -36,36 +47,33 @@ particular emphasis is placed on code quality and reproducibility.
...
+conversion through to actual QMC runs.
 
 # Building with CMake
 
- The build system for QMCPACK is based on CMake.  It will auto-configure based on the detected compilers and libraries. Previously
- QMCPACK made extensive use of toolchains, but the system has since been updated to eliminate the use of toolchain files for most
- cases.  Specific compile options can be specified either through specific environment or CMake variables.  When the libraries are
- installed in standard locations, e.g., /usr, /usr/local, there is no need to set environment or CMake variables for the packages.
+ The build system for QMCPACK is based on CMake.  It will auto-configure based on the detected compilers and libraries. Specific
+ compile options can be specified either through specific environment or CMake variables.  When the libraries are installed in
```
Comment (markdewing) [inline README.md:75]: The sentence 'Specific compile options ...' is a little unclear to me.  In the previous version, it's contrasting with toolchains, but now that the sentence about toolchains was removed, I'm not sure what it is trying to convey.

**Comment by markdewing** [comment]
Test this please

**Pull request merged**